lib/heroku-scaler.rb

```diff
@@ -0,0 +1 @@
+require 'heroku-scaler/railtie' if defined?(Rake)
```

Comment:
You could probably have it not blow up outside of Rails if you add `&& defined?(Rails)` here, but then the Rake tasks themselves probably wouldn't be available to the including application.